views/new.ejs

```diff
@@ -5,12 +5,12 @@
   <body>
     <div class="card">
       <form action="/books" method="POST" id="newBook">
-        <label>Title<input type="text" name="title" required /></label>
-        <label>Author<input type="text" name="author" required /></label>
-        <label>ISBN<input type="text" name="isbn" required /></label>
-        <label>Image_URL<input type="text" name="image_url" required /></label>
-        <label>Description<textarea form="newBook" name="description" required></label>
-        <input type="submit">
+        <label>Title<input class="input-field" type="text" name="title" required /></label>

```

Comment:
you probably don't want all these input fields to be `required` in case you want to add a book but don't necessarily have all the info. At a minimum, i would probably only require author and title.